pyproject.toml

```
@@ -164,7 +164,7 @@ test = [
     "hdbscan>=0.8.33",  # Previous version had a broken wheel
 
     # for sortingview backend
-    "sortingview",
+    "sortingview<0.14.0",
```

That's done @zm711

---

```suggestion
    "sortingview>=0.12.0,<0.14.0",
```
I was saying this? Do we need the lower limit like we have in above so that we don't allow people to build the broken one.

---

Ehehehe you're right! Let me add that!